Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1704.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.CustomAttributes;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 1704, "[Enhancement] Basic gif animation features", PlatformAffected.UWP)]
+	public class Issue1704 : TabbedPage
+	{
```

Comment:
Can you create a UI Test that just loads this issue? to at least ensure that nothing crashes?

Reply:
@PureWeen, I have not done a UI test before (just issue tests) so don't know details yet. Should I add a new class into Xamarin.Forms.Core.UITests and hook up my issue class using similar mechanics as ImageUITest and add something into the ControlGalleryPages similar to ImageCoreGalleryPage.cs or is there some easier way just to reuse my issue class? How do I run the UI tests ones added?

Reply:
Here's a sample for a basic UITEST. You just change the structure a little bit of the test and add an IFDEF for the UI Test.  The Controls.ISsues.Shared is part of the UI test project as well

https://github.com/xamarin/Xamarin.Forms/blob/122f0e3e6060711d6c3b2b8523236b8a057434df/Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3292.cs

and then I would also add
```C#
[Category(UITestCategories.ManualReview)]
```

Since it'll require someone to physically look at the gifs to make sure they are playing

> How do I run the UI tests once added?
- Deploy the Control Gallery to whatever device you want to run the UI Test on
- Now compile whatever project in the *UITests* folder you want to run 
- at this point the UI Test you added should show up in the text explorer and you should just be able to run it and it will run the UI Test

Reply:
@PureWeen Added UITest for Issue1704.